cmd/odf/get/dr_prereq.go

```diff
@@ -0,0 +1,18 @@
+package get
+
+import (
+	"github.com/red-hat-storage/odf-cli/pkg/odf/dr"
+	"github.com/spf13/cobra"
+)
+
+var drPrereqCmd = &cobra.Command{
+	Use:                "dr-prereq",
+	Short:              "Print the status of pre-requisites for Disaster Recovery between peer clusters.",
+	DisableFlagParsing: true,
+	Args:               cobra.ExactArgs(1),
+	Example:            "odf get dr-prereq <peer-clusterID>",
```

Comment:
In the readme the arg is called a cluster name, rather than an ID. Should one be renamed for consistency?

Reply:
Agreed. Changing this to `<PeerManagedClusterName>`